core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -901,4 +903,32 @@ static function (array $allowedProperties): array {
             ->end()
         ;
     }
+
+    private function addAltchaNode(): NodeDefinition
+    {
+        return (new TreeBuilder('altcha'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->enumNode('algorithm')
+                    ->info('The algorithm used to generate the challenges. Select between "SHA-256", "SHA-384" or "SHA-512".')
+                    ->values(Algorithm::values())
+                    ->defaultValue(Algorithm::sha256->value)
+                ->end()
+                ->integerNode('range_min')
+                    ->info('A higher value increases the complexity/security but may significantly increase the computational load on client devices, potentially impacting the user experience.')
+                    ->defaultValue(10000)
+                ->end()
```

Comment:
This is not correct I think. A higher `range_min` actually decreases the complexity because the range gets smaller. I don’t see the reason why we would need this at all. The documentation says _“We recommend using a minimum value of the random number of at least several thousand as a “minimum required complexity”. ”_ but I don’t understand it why (yet) 🙈

Reply:
ALTCHA does not use a minnumber in their own implementation see https://github.com/altcha-org/altcha-lib/blob/c1f2b989c47685ee84fa6ce0c4b11d7de76d07a7/lib/index.ts#L40 so I think we can safely remove `range_min` here.

Reply:
It is used here:

<img width="648" alt="" src="https://github.com/contao/contao/assets/1192057/fab37e00-4a74-4c65-9eb6-fe5c60586faa">

Reply:
It is used here:

<img width="648" alt="" src="https://github.com/contao/contao/assets/1192057/f2db92bf-1f99-4ed8-8115-036dde05b134">

Reply:
Yes, but we should use `$number ??= random_int(0, $this->altchaRangeMax);` instead. I will create an issue about that in the ALTCHA repo to make sure I’m not misinterpreting it.

Reply:
https://github.com/altcha-org/altcha/issues/41

Reply:
Removed in 170a386e42673218efb798fd1611a3434d318791.